exchange/wrappers/buzzex.js

```diff
@@ -0,0 +1,335 @@
+const Buzzex = require('buzzex-gekko-npm');
+const moment = require('moment');
+const _ = require('lodash');
+const exchangeUtils = require('../exchangeUtils');
+const retry = exchangeUtils.retry;
+const scientificToDecimal = exchangeUtils.scientificToDecimal;
+
+const marketData = require('./buzzex-markets.json');
+
+const Trader = function(config) {
+  _.bindAll(this);
+
+  if(_.isObject(config)) {
+    this.key = config.key;
+    this.secret = config.secret;
+    this.currency = config.currency.toUpperCase()
+    this.asset = config.asset.toUpperCase();
+  }
+
+  this.name = 'Buzzex';
+  this.since = null;
+  
+  this.market = _.find(Trader.getCapabilities().markets, (market) => {
+    return market.pair[0] === this.currency && market.pair[1] === this.asset
+  });
+  this.pair = this.market.book;
+
+  this.interval = 3100;
+
+  this.buzzex = new Buzzex(
+    this.key,
+    this.secret,
+    {timeout: +moment.duration(60, 'seconds')}
+  );
+}
+
+const recoverableErrors = [
+  'SOCKETTIMEDOUT',
+  'TIMEDOUT',
+  'CONNRESET',
+  'CONNREFUSED',
+  'NOTFOUND',
+  'Service:Unavailable',
+  'Request timed out',
+  'Empty response',
+  'API:Invalid nonce',
```

Comment:
This is exactly the same as what Kraken sends. Unless this exchange actually replies with that error, can you take it out?